### PR TITLE
add tcpdump and needed libpcap dependency

### PR DESCRIPTION
**Pull request by abbbi**
## Description of package

libpcap and tcpdump

## Installed manifest of package

libpcap:
```
/var/db/kiss/installed/libpcap/version
/var/db/kiss/installed/libpcap/sources
/var/db/kiss/installed/libpcap/manifest
/var/db/kiss/installed/libpcap/depends
/var/db/kiss/installed/libpcap/checksums
/var/db/kiss/installed/libpcap/build
/var/db/kiss/installed/libpcap/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/share/man/man7/pcap-tstamp.7
/usr/share/man/man7/pcap-linktype.7
/usr/share/man/man7/pcap-filter.7
/usr/share/man/man7/
/usr/share/man/man5/pcap-savefile.5
/usr/share/man/man5/
/usr/share/man/man3/pcap_tstamp_type_val_to_name.3pcap
/usr/share/man/man3/pcap_tstamp_type_val_to_description.3pcap
/usr/share/man/man3/pcap_tstamp_type_name_to_val.3pcap
/usr/share/man/man3/pcap_strerror.3pcap
/usr/share/man/man3/pcap_statustostr.3pcap
/usr/share/man/man3/pcap_stats.3pcap
/usr/share/man/man3/pcap_snapshot.3pcap
/usr/share/man/man3/pcap_setnonblock.3pcap
/usr/share/man/man3/pcap_setfilter.3pcap
/usr/share/man/man3/pcap_setdirection.3pcap
/usr/share/man/man3/pcap_set_tstamp_type.3pcap
/usr/share/man/man3/pcap_set_tstamp_precision.3pcap
/usr/share/man/man3/pcap_set_timeout.3pcap
/usr/share/man/man3/pcap_set_snaplen.3pcap
/usr/share/man/man3/pcap_set_rfmon.3pcap
/usr/share/man/man3/pcap_set_protocol_linux.3pcap
/usr/share/man/man3/pcap_set_promisc.3pcap
/usr/share/man/man3/pcap_set_immediate_mode.3pcap
/usr/share/man/man3/pcap_set_datalink.3pcap
/usr/share/man/man3/pcap_set_buffer_size.3pcap
/usr/share/man/man3/pcap_sendpacket.3pcap
/usr/share/man/man3/pcap_perror.3pcap
/usr/share/man/man3/pcap_open_offline_with_tstamp_precision.3pcap
/usr/share/man/man3/pcap_open_offline.3pcap
/usr/share/man/man3/pcap_open_live.3pcap
/usr/share/man/man3/pcap_open_dead_with_tstamp_precision.3pcap
/usr/share/man/man3/pcap_open_dead.3pcap
/usr/share/man/man3/pcap_offline_filter.3pcap
/usr/share/man/man3/pcap_next_ex.3pcap
/usr/share/man/man3/pcap_next.3pcap
/usr/share/man/man3/pcap_minor_version.3pcap
/usr/share/man/man3/pcap_major_version.3pcap
/usr/share/man/man3/pcap_loop.3pcap
/usr/share/man/man3/pcap_lookupnet.3pcap
/usr/share/man/man3/pcap_lookupdev.3pcap
/usr/share/man/man3/pcap_list_tstamp_types.3pcap
/usr/share/man/man3/pcap_list_datalinks.3pcap
/usr/share/man/man3/pcap_lib_version.3pcap
/usr/share/man/man3/pcap_is_swapped.3pcap
/usr/share/man/man3/pcap_inject.3pcap
/usr/share/man/man3/pcap_getnonblock.3pcap
/usr/share/man/man3/pcap_geterr.3pcap
/usr/share/man/man3/pcap_get_tstamp_precision.3pcap
/usr/share/man/man3/pcap_get_selectable_fd.3pcap
/usr/share/man/man3/pcap_get_required_select_timeout.3pcap
/usr/share/man/man3/pcap_freecode.3pcap
/usr/share/man/man3/pcap_freealldevs.3pcap
/usr/share/man/man3/pcap_free_tstamp_types.3pcap
/usr/share/man/man3/pcap_free_datalinks.3pcap
/usr/share/man/man3/pcap_fopen_offline_with_tstamp_precision.3pcap
/usr/share/man/man3/pcap_fopen_offline.3pcap
/usr/share/man/man3/pcap_findalldevs.3pcap
/usr/share/man/man3/pcap_fileno.3pcap
/usr/share/man/man3/pcap_file.3pcap
/usr/share/man/man3/pcap_dump_open.3pcap
/usr/share/man/man3/pcap_dump_ftell.3pcap
/usr/share/man/man3/pcap_dump_fopen.3pcap
/usr/share/man/man3/pcap_dump_flush.3pcap
/usr/share/man/man3/pcap_dump_file.3pcap
/usr/share/man/man3/pcap_dump_close.3pcap
/usr/share/man/man3/pcap_dump.3pcap
/usr/share/man/man3/pcap_dispatch.3pcap
/usr/share/man/man3/pcap_datalink_val_to_name.3pcap
/usr/share/man/man3/pcap_datalink_val_to_description_or_dlt.3pcap
/usr/share/man/man3/pcap_datalink_val_to_description.3pcap
/usr/share/man/man3/pcap_datalink_name_to_val.3pcap
/usr/share/man/man3/pcap_datalink.3pcap
/usr/share/man/man3/pcap_create.3pcap
/usr/share/man/man3/pcap_compile.3pcap
/usr/share/man/man3/pcap_close.3pcap
/usr/share/man/man3/pcap_can_set_rfmon.3pcap
/usr/share/man/man3/pcap_breakloop.3pcap
/usr/share/man/man3/pcap_activate.3pcap
/usr/share/man/man3/pcap.3pcap
/usr/share/man/man3/
/usr/share/man/man1/pcap-config.1
/usr/share/man/man1/
/usr/share/man/
/usr/share/
/usr/lib/pkgconfig/libpcap.pc
/usr/lib/pkgconfig/
/usr/lib/libpcap.so.1.9.1
/usr/lib/libpcap.so.1
/usr/lib/libpcap.so
/usr/lib/libpcap.a
/usr/lib/
/usr/include/pcap/vlan.h
/usr/include/pcap/usb.h
/usr/include/pcap/socket.h
/usr/include/pcap/sll.h
/usr/include/pcap/pcap.h
/usr/include/pcap/pcap-inttypes.h
/usr/include/pcap/nflog.h
/usr/include/pcap/namedb.h
/usr/include/pcap/ipnet.h
/usr/include/pcap/funcattrs.h
/usr/include/pcap/dlt.h
/usr/include/pcap/compiler-tests.h
/usr/include/pcap/can_socketcan.h
/usr/include/pcap/bpf.h
/usr/include/pcap/bluetooth.h
/usr/include/pcap/
/usr/include/pcap.h
/usr/include/pcap-namedb.h
/usr/include/pcap-bpf.h
/usr/include/
/usr/bin/pcap-config
/usr/bin/
/usr/
```

tcpdump:
```
/var/db/kiss/installed/tcpdump/version
/var/db/kiss/installed/tcpdump/sources
/var/db/kiss/installed/tcpdump/manifest
/var/db/kiss/installed/tcpdump/depends
/var/db/kiss/installed/tcpdump/checksums
/var/db/kiss/installed/tcpdump/build
/var/db/kiss/installed/tcpdump/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/share/man/man1/tcpdump.1
/usr/share/man/man1/
/usr/share/man/
/usr/share/
/usr/sbin/tcpdump
/usr/sbin/
/usr/

```
## New package

- [ x] Latest upstream version.
- [ ] I agree to maintain this package.
- [ ] I agree to notify @dylanaraps if I can no longer maintain this package.

## Existing package

- [ ] I am the maintainer of this package.

### CODE REVIEW

File: community/tcpdump/depends [hunk 1]
```diff
@@ -0,0 +1,2 @@
+make
```
Comment (dylanaraps) [inline community/tcpdump/depends:1]: Make is an implicit dependency and doesn't need to be listed.

File: community/libpcap/build [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#!/bin/sh -e
+
+./configure \
+    --prefix=/usr
+
```
Comment (dylanaraps) [inline community/libpcap/build:5]:
```sh

make
```

Comment (dylanaraps) [reply community/libpcap/build:5]: Even if unneeded we run `make` prior to `make install`.

File: community/tcpdump/build [hunk 3]
```diff
@@ -0,0 +1,9 @@
+#!/bin/sh -e
+
+./configure \
+    --prefix=/usr
+
+make DESTDIR="$1" install
```
Comment (dylanaraps) [inline community/tcpdump/build:6]: Same as my other comment.

File: community/tcpdump/build [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+
+# remove duplicate versioned executable
+rm -f "$1/usr/sbin/tcpdump.4.9.3"
```
Comment (dylanaraps) [inline community/tcpdump/build:9]: Is this a symlink or a real duplicate?

Comment (abbbi) [reply community/tcpdump/build:9]: Its a real duplicate.

File: community/tcpdump/build [hunk 5]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh -e
+
+./configure \
+    --prefix=/usr
+
+make
+make DESTDIR="$1" install
+
+# remove duplicate versioned executable
```
Comment (dylanaraps) [inline community/tcpdump/build:9]:
```sh
# Remove duplicate executable.
````

**Comment by dylanaraps** [comment]
`tcpdump` needs to install to `/usr/bin` (not `sbin`). Also for next time, one PR per package please!

**Comment by dylanaraps** [comment]
:+1:

**Pull request merged**